src/App.jsx

```diff
@@ -14,14 +15,23 @@ function App() {
       "https://vignette.wikia.nocookie.net/mrmen/images/7/78/Mr-Grumpy-3A.PNG/revision/latest?cb=20170707233013",
   });
 
+  const [reviewId, setReviewId] = useState(1);
+
   return (
     <BrowserRouter>
       <main>
         <Header />
         <Nav />
         <Routes>
           <Route path="/" element={<CategoriesList />} />
-          <Route path="/reviews" element={<ReviewsList />} />
+          <Route
+            path="/reviews"
+            element={<ReviewsList setReviewId={setReviewId} />}
+          />
+          <Route
+            path="/reviews/review"
```

Comment:
The path should be something like "/reviews/:review_id" at the moment, every article is under the same url 🙂 
Once you modify make sure that articles' link look like /reviews/12